src/components/date-picker-day/date-picker-day.e2e.ts

```diff
@@ -1,14 +1,20 @@
 import { disabled } from "../../tests/commonTests";
 import { newProgrammaticE2EPage } from "../../tests/utils";
+import { numberStringFormatter } from "../../utils/locale";
 
 describe("calcite-date-picker-day", () => {
   it("can be disabled", async () => {
+    numberStringFormatter.numberFormatOptions = {
```

Comment:
Or I could just return the `localize`/`delocalize` methods' params if `numberFormatOptions` isn't set, which would effectively assume `lang === "en && numberingSystem === "latn"`. e.g.

```js
localize = (numberString: string) =>
    Object.keys(this._numberFormatOptions).length
      ? doFormattyStuff()
      : numberString;
```

It would be the equivolent outcome as adding a constructor and actually populating the internal properties, but without the extra formatter creations.

Reply:
I went with my second comment :point_up: 

Now `numberStringFormatter.numberFormatOptions` only needs to be set when the `lang` and `numberingSystem` props are not their default values ("en" and "latn" respectively). This means that for a good portion of our users, we won't be initializing a formatter at all.

That's down from 10+ formatters created/destroyed for each character typed in `calcite-input` alone :rocket: